doc/release-notes/RELEASE-1.0.md

```diff
@@ -0,0 +1,29 @@
+# Release 1.0
+
+We are pleased to announce the release of Screed v0.9. Screed is a database
+engine capable of storing and retrieving short-read sequence data and is
+designed to be fast and adaptable to different sequence file formats.
```

Comment:
This paragraph is marked 0.9 but has better (and newer?) language. Duplicated paragraph below?